src/main/java/com/github/jsonapi/retrofit/JSONAPIConverterFactory.java

```diff
@@ -18,6 +18,11 @@
 public class JSONAPIConverterFactory extends Converter.Factory {
 	private ResourceConverter parser;
 
+	public JSONAPIConverterFactory(ResourceConverter parser)
+	{
```

Comment:
For sakes of consistency, can you please put the `{` inline with the constructor declaration rather than having it in its own line.